### PR TITLE
UIINREACH-219 - INN-Reach paging slips not breaking pages between slips

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
## Purpose
INN-Reach paging slips not breaking pages between slips

## Refs
https://issues.folio.org/browse/UIINREACH-219

## Screenshots
o43EFHOcVT

### CODE REVIEW

File: src/components/common/SearchAndFilter/components/PrintSlips/PrintSlips.js [hunk 1]
```diff
@@ -45,15 +45,20 @@ class PrintSlips extends React.Component {
     const hasTemplates = !isEmpty(templates);
 
     const renderTemplates = () => {
-      return templatesContext.map((templateContext) => {
+      return templatesContext.map((templateContext, idx) => {
         const templateWithBrake = templates[templateContext['innReachTransaction.centralServerId']].concat('</br>');
```
Comment (Dmytro-Melnyshyn) [inline src/components/common/SearchAndFilter/components/PrintSlips/PrintSlips.js:49]: Is `</br>` still needed?

File: src/components/common/SearchAndFilter/components/PrintSlips/PrintSlips.js [hunk 2]
```diff
@@ -45,15 +45,20 @@ class PrintSlips extends React.Component {
...
-            key={templateContext['innReachTransaction.centralServerId']}
-            template={templateWithBrake}
-            dataSource={templateContext}
-          />
+          <div
+            key={idx}
+            style={{ pageBreakAfter: 'always' }}
+          >
+            <ComponentToPrint
+              key={templateContext['innReachTransaction.centralServerId']}
```
Comment (Dmytro-Melnyshyn) [inline src/components/common/SearchAndFilter/components/PrintSlips/PrintSlips.js:57]: The `key` prop is no longer needed.

File: src/components/common/SearchAndFilter/components/PrintSlips/PrintSlips.js [hunk 3]
```diff
@@ -45,15 +45,20 @@ class PrintSlips extends React.Component {
...
 
         return (
-          <ComponentToPrint
-            key={templateContext['innReachTransaction.centralServerId']}
-            template={templateWithBrake}
-            dataSource={templateContext}
-          />
+          <div
+            key={idx}
+            style={{ pageBreakAfter: 'always' }}
```
Comment (Dmytro-Melnyshyn) [inline src/components/common/SearchAndFilter/components/PrintSlips/PrintSlips.js:54]: Please use the `className` instead of the `style`.

**Review by BogdanDenis — APPROVED** [review]

**Comment by Terala-Priyanka** [comment]
@Dmytro-Melnyshyn 
Addressed all your comments.

File: src/components/common/SearchAndFilter/components/PrintSlips/PrintSlips.js [hunk 4]
```diff
@@ -46,15 +46,14 @@ class PrintSlips extends React.Component {
 
     const renderTemplates = () => {
       return templatesContext.map((templateContext, idx) => {
-        const templateWithBrake = templates[templateContext['innReachTransaction.centralServerId']].concat('</br>');
+        const templateWithBrake = templates[templateContext['innReachTransaction.centralServerId']];
```
Comment (Dmytro-Melnyshyn) [inline src/components/common/SearchAndFilter/components/PrintSlips/PrintSlips.js:49]: If there is no need for a break here, let's just rename the variable to `template`.

**Review by Dmytro-Melnyshyn — APPROVED** [review]

File: src/components/common/SearchAndFilter/components/PrintSlips/PrintSlips.js (hunk 4 above)
Comment (Terala-Priyanka) [reply src/components/common/SearchAndFilter/components/PrintSlips/PrintSlips.js]: Thank you

**Pull request merged**